src/components/Facility/ConsultationDoctorNotes/index.tsx

```diff
@@ -153,47 +153,50 @@ const ConsultationDoctorNotes = (props: ConsultationDoctorNotesProps) => {
             </button>
           ))}
         </div>
-        <PatientConsultationNotesList
-          state={state}
-          setState={setState}
-          reload={reload}
-          setReload={setReload}
-          thread={thread}
-          setReplyTo={setReplyTo}
-        />
-        <DoctorNoteReplyPreviewCard
-          parentNote={reply_to}
-          cancelReply={() => setReplyTo(undefined)}
-        >
-          <div className="relative mx-4 flex items-center">
-            <AutoExpandingTextInputFormField
-              id="doctor_consultation_notes"
-              maxHeight={160}
-              rows={2}
-              name="note"
-              value={noteField}
-              onChange={(e) => setNoteField(e.value)}
-              className="w-full grow"
-              innerClassName="pr-10"
-              errorClassName="hidden"
-              placeholder={t("notes_placeholder")}
-              disabled={!patientActive}
-              onFocus={() => setFocused(true)}
-              onBlur={() => setFocused(false)}
+        <div className="flex flex-1 overflow-hidden">
+          <div
+            className={classNames(
+              "flex flex-1 flex-col",
+              threadViewNote && "max-sm:hidden",
+            )}
+          >
+            <PatientConsultationNotesList
+              state={state}
+              setState={setState}
+              reload={reload}
+              setReload={setReload}
+              thread={thread}
+              setReplyTo={setReplyTo}
+              mode={mode}
+              setThreadViewNote={setThreadViewNote}
             />
-            <ButtonV2
-              onClick={onAddNote}
-              border={false}
-              className="absolute right-2"
-              ghost
-              size="small"
-              disabled={!patientActive}
-              authorizeFor={NonReadOnlyUsers}
-            >
-              <CareIcon icon="l-message" className="text-lg" />
-            </ButtonV2>
+            <div className="mt-2">
+              <DoctorNoteReplyPreviewCard
+                parentNote={reply_to}
+                cancelReply={() => setReplyTo(undefined)}
+              >
+                <DiscussionNotesEditor
+                  initialNote={noteField}
+                  onChange={setNoteField}
+                  onAddNote={onAddNote}
+                  isAuthorized={patientActive}
+                  onRefetch={() => setReload(true)}
```

Comment:
Double check the logic here, in current logic, reload will not happen when the value is true
```suggestion
                  onRefetch={(prev) => setReload(!prev)}
```

Also, if you are trying to trigger an API call, useQuery returns refetch function, consider that

Reply:
resolved